internal/core/common/ecs/ecs.go

```diff
@@ -0,0 +1,40 @@
+package ecs
+
+import "strings"
+
+type TaskMetadata struct {
+	ClusterName	string `json:"Cluster"`
+	TaskARN		string `json:"TaskARN"`
+	Family		string `json:"Family"`
+	Revision	string `json:"Revision"`
+	KnownStatus	string `json:"KnownStatus"`
+	Containers  []Container `json:"Containers"`
+}
+
+func (task *TaskMetadata) GetDimensions() map[string]string {
```

Comment:
We also need to figure out whether it is appropriate to send an `AWSUniqueId` dimension with this or not.  Another question is whether the agent is going to discover the AWSUniqueId of the underlying host or not.  Do you see AWSUniqueId as a dimension already on stuff you have been testing with? If so, we might need to disable the auto determination of it from the EC2 metadata endpoint.